README.md

```diff
@@ -10,7 +10,15 @@ be used in production systems.
 ## Getting started
 - First install libsnark's dependencies as specified [here](https://github.com/scipr-lab/libsnark#dependencies)
 - Then, make sure you have [opam](https://opam.ocaml.org/doc/Install.html) installed.
-- Then, install `snarky` by running
+- Next, install snarky's dependencies from the repo
+```bash
+opam pin add fold_lib git@github.com:o1-labs/snarky.git
+opam pin add tuple_lib git@github.com:o1-labs/snarky.git
+opam pin add bitstring_lib git@github.com:o1-labs/snarky.git
+opam pin add interval_union git@github.com:o1-labs/snarky.git
+opam pin add ppx_snarky git@github.com:o1-labs/snarky.git
+```
```

Comment:
this is unfortunate :/ . Is there no way for `opam pin add snarky` to get dependencies for us?

Reply:
Perhaps we should set up the snarky repo as an opam repo, then that just becomes
```bash
opam respository add snarky git@github.com:o1-labs/snarky.git
opam install snarky
```